lib/Support/Unix/Threading.inc

```diff
@@ -142,6 +142,7 @@ void llvm::set_thread_name(const Twine &Name) {
   if (get_max_thread_name_length() > 0)
     NameStr = NameStr.take_back(get_max_thread_name_length() - 1);
   (void)NameStr;
+#if 0
```

Comment:
Is it possible to set HAVE_PTHREAD_SETNAME_NP to zero (whatever) from the cmake command line?

Reply:
Yes, it is possible

Reply:
Another option is setting these macros in the cmake files instead of cmake command line.
```
set(HAVE_PTHREAD_SETNAME_NP 0)
set(HAVE_PTHREAD_GETNAME_NP 0)

```

Reply:
It's a little counter-intuitive to set an auto-detected property in the CMakeLists.txt file.  It might be better to add yet another macro name (not starting with `HAVE_`) to forcibly disable this feature.

Reply:
Someone who is interested could change the #if 0 to be based on a compile-line define instead, which could be set by the CMakefile depending on HAVE_PTHREAD_SETNAME_NP.  I don't think setting HAVE_PTHREAD_SETNAME_NP should be in the flang CMakefile.